### ISSUE TITLE
 Support for @scoped packages

### ISSUE THREAD

**Issue by mikerandall** [willdurand/puppet-nodejs#138]
### Description

Need the ability to sign in and retrieve privately scoped NPM modules.   A username and password field on the package would work, but it might be nice to use an .npmrc file that would be placed in the image.

**Comment by grafjo** [comment]
@michaelbeil have you tried ~/.npmrc?

**Comment by mikerandall** [comment]
Yes, but that isn't the problem. There's no way to download scoped packages, and no way to sign in as a user with the current configuration. As more people stop using Bower, scoped packages are going to become more popular.

**Comment by Ma27** [comment]
According to the docs (https://docs.npmjs.com/misc/scope) it should be possible to do this with some npm commands. It should be possible to do this with the usage of "exec", but actually you're right, such a feature should exist in 2.0 :)

**Comment by Ma27** [comment]
postponing to 2.1: 2.0 contains some important releases and it would be nice to get a release in the next time

**Comment by Ma27** [comment]
@mikerandall just so you know - I still know about this feature request, I'm unfortunately way to busy with different things, so I'm currently unable to do active feature development for `puppet-nodejs` (sorry about that!) I don't know if you're still in need for this, but I don't think that I'll be able to do this in near time :(

It would be totally awesome if you could file a patch, but no rush, we'll see if I have time for this in the future :)
